player/configfiles.c

```diff
@@ -215,7 +215,7 @@ static char *mp_get_playback_resume_config_filename(struct MPContext *mpctx,
     char *res = NULL;
     void *tmp = talloc_new(NULL);
     const char *path = NULL;
-    if (opts->ignore_path_in_watch_later_config && !mp_is_url(bstr0(path))) {
+    if (opts->ignore_path_in_watch_later_config && !mp_is_url(bstr0(fname))) {
```

Comment:
I think `mp_normalize_path` does nothing for urls, so this looks like the url check should be separate?

Reply:
Do you mean

```c
    const char *path = NULL;
    if (mp_is_url(bstr0(fname))) {
        path = fname;
    } else if (opts->ignore_path_in_watch_later_config) {
        path = mp_basename(fname);
    } else {
        path = mp_normalize_path(tmp, fname);
        if (!path)
            goto exit;
    }
```

? It's just 2 fewer lines to let `mp_normalize_path` do nothing to URLs.

Reply:
Yes. This shows what's going on and avoids pointless copy of path.